sum-of-nth-terms.py

```diff
@@ -0,0 +1,15 @@
+def series_sum(n):
+    if n == 0:
+        return "%.2f" % n
+    elif n == 1:
+        return "%.2f" % n
+    else:
```

Comment:
Since both of the above conditionals end in `return` statements, this `else` isn't necessary.